src/mapping_navigation/scripts/temp_car_control.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
Is this a temporary file? Think that the steering information goes to the central control, which will also deal with throttling

Reply:
Yeah it is, its just for convenience, will be removed before pushing